docs/en/operations/settings/settings_users.md

```diff
@@ -0,0 +1,125 @@
+# User settings
+
+The `users` section of the `user.xml` configuration file aggregates settings for users.
+
+Structure of the `users` section:
+
+```
+<users>
+    <!-- If user name was not specified, 'default' user is used. -->
+    <user_name>
+        <password></password>
+        <!-- Or -->
+        <password_sha256_hex></password_sha256_hex>
+
+        <networks incl="networks" replace="replace">
+        </networks>
+
+        <profile>profile_name</profile>
+
+        <quota>default</quota>
+
+        <databases>
+            <database_name>
+                <table_name>
+                    <filter>expression</filter>
+                <table_name>
+            </database_name>
+        </databases>
+    </user_name>
+    <!-- Other users settings -->
+</users>
+```
+
+### user_name/password
+
+Password could be specified in plaintext or in SHA256 (hex format).
+
+- To specify password in plaintext (not recommended), place it in a `password` element.
+
+    For example, `<password>qwerty</password>`. Password can be empty.
+
+- To specify SHA256 hash of a password, place it in a `password_sha256_hex` element.
+
+    For example, `<password_sha256_hex>65e84be33532fb784c48129675f9eff3a682b27168c0ea744b2cf58ee02337c5</password_sha256_hex>`.
+
+    To generate password, in the command prompt perform the command:
+
+    ```
+    PASSWORD=$(base64 < /dev/urandom | head -c8); echo "$PASSWORD"; echo -n "$PASSWORD" | sha256sum | tr -d '-'
+    ```
+
+    The first line of the result is the password. The second line is the corresponding SHA256 hash.
+
+
+### user_name/networks
+
+List of networks which the user can connect to ClickHouse server from.
+
+Each element of list has one of the following forms:
+
+- `<ip>` — IP-address or a network mask.
```

Comment:
we could explicitly say that either IPv4 and IPv6 are supported, it's still not that common

Reply:
There is an example that shows it. Or I understood your comment wrong?